### PR TITLE
Bill Situ <Bill.Situ@oracle.com>

### PR DESCRIPTION
**Pull request by bsitu**
On branch cna/Oracle/CVE-2020-14729v2
 Changes to be committed:
	modified:   2020/14xxx/CVE-2020-14729.json

### CODE REVIEW

File: 2020/14xxx/CVE-2020-14729.json [hunk 1]
```diff
@@ -62,7 +62,7 @@
         "reference_data": [
```
Comment (cve-team) [inline 2020/14xxx/CVE-2020-14729.json:62]: MISC or CONFIRM, but name != url for reference.

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2020/14xxx/CVE-2020-14729.json (hunk 1 above)
Comment (cristina479) [reply 2020/14xxx/CVE-2020-14729.json:62]: The `name` and `url` fields need to have the same value for `MISC` or `CONFIRM` references.

**Review by ghost — APPROVED** [review]

File: 2020/14xxx/CVE-2020-14729.json [hunk 2]
```diff
@@ -62,7 +62,7 @@
...
             {
                 "url": "https://system.netsuite.com/app/help/helpcenter.nl",
-                "name": "https://system.netsuite.com/app/help/helpcenter.nl",
+                "name": "Remediation is provided at customer visible site",
```
Comment (ghost) [inline 2020/14xxx/CVE-2020-14729.json:65]:
```suggestion
                "name": "Remediation is provided at customer visible site",
```
```suggestion
                "name": "Remediation is provided at customer visible site",
```

**Review by ghost — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Review by Rayyan335 — APPROVED** [review]

**Review by Rayyan335 — APPROVED** [review]

**Comment by bsitu** [comment]
This has been resolved.

**Pull request closed without merging**